cg/constants/nextflow.py

```diff
@@ -12,6 +13,19 @@
 NXF_PID_FILE_ENV = "NXF_PID_FILE"
 NXF_JVM_ARGS_ENV = "NXF_JVM_ARGS"
 JAVA_MEMORY_HEADJOB = "-Xmx5g"
+NEXTFLOW_WORKFLOWS = [
+    Workflow.RAREDISEASE,
+    Workflow.RNAFUSION,
+    Workflow.TAXPROFILER,
+    Workflow.TOMTE,
+]
+NEXTFLOW_WORKFLOWS_DEV = [
```

Comment:
Do we need this one? If it not used, we should not add it. Can't a workflow being added just add itself to the relevant tests as `Workflow.XYZ` and then once fully integrated add itself to `NEXTFLOW_WORKFLOWS `? Bundling them together requires to add everything in one PR to not brake the tests. What issue will be solved by this constant? Maybe I am missing something?